CHANGELOG.md

```diff
@@ -7,10 +7,12 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [2.16.0] - 2021-06-01
+
 ### Changed
 
-- Use host network, so chart-operator can install CNI packaged as an app.
-- Add tolerations to start on `NotReady` nodes.
+- For CAPI clusters use host network, so chart-operator can install CNI packaged as an app.
+- For CAPI clusters add tolerations to start on `NotReady` nodes for installing CNI.
```

Comment:
I added this because for non-CAPI clusters nothing changes.

This will be enabled via the cluster configmap managed by cluster-apps-operator.